packages/react-router/modules/RouterContext.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const RouterContext = React.unstable_createContext(0);
```

Comment:
Not sure if `{}` is a valid default value, but it would be better.

Reply:
I believe it should be the shape of the expected context. Something like this, but I'm not sure how specific it is supposed to be.

```js
React.unstable_createContext({
 history: null,
 route: {
   location: null,
   match: null
  }
});
```
 The `<StaticRouter>` also places a `staticContext` on the context, but whether it is necessary to include it I cannot say.

Reply:
It can be whatever you want it to be. `null` seems like an appropriate choice.